docs/configuration/hyperparameter_optimization.md

```diff
@@ -193,26 +137,24 @@ hyperopt:
         - [{"output_size": 512}, {"output_size": 256}]
         - [{"output_size": 512}]
         - [{"output_size": 256}]
-  goal: minimize
 ```
 
-# Executor
+# Search Algorithm
 
-## Serial Executor
-
-The `serial` executor performs hyperparameter optimization locally in a serial manner, executing the elements in the set
-of sampled parameters obtained by the selected sampler one at a time.
-
-Example:
+Ray Tune supports its own collection of [search algorithms](https://docs.ray.io/en/master/tune/api_docs/suggestion.html), specified by the `search_alg` section of the hyperopt config:
 
 ```yaml
-executor:
-  type: serial
+search_alg:
+  type: variant_generator
 ```
 
+You can find the full list of supported search algorithm names in Ray Tune's [create_searcher](https://github.com/ray-project/ray/blob/master/python/ray/tune/suggest/__init__.py) function. Please note these algorithms require installation of additional packages.  As of this version of Ludwig, Ludwig installs the packages for the search algorithm `hyperopt`.  For all other search algorithms, the user is expected to install the required packages.
+
+# Executor
+
 ## Ray Tune Executor
 
-The `ray` executor is used in conjunction with the `ray` sampler to enable [Ray Tune](https://docs.ray.io/en/master/tune/index.html) for distributed hyperopt across a cluster of machines.
+The `ray` executor is used to enable [Ray Tune](https://docs.ray.io/en/master/tune/index.html) for both local and distributed hyperopt across a cluster of machines.
 
 **Parameters:**
```

Comment:
Need to add num_samples here and explain the logic wwhen mixing grid and other spaces